modules/bmc_check/esbmccheck.py

```diff
@@ -34,30 +36,29 @@ def __init__(self):
         self.maxdepthverification = 25
         self.en_kparalell = False
         self.esbmcpath = ''
-        self.esbmc_arch = "--64"
         self.esbmc_bound = 1
-        self.esbmc_unwind_op = "--unwind"
+        self.esbmc_unwind_op = "  --state-hashing --unwind"
         self.esbmc_memlimit_op = ""
         self.esbmc_timeout_op = "15m"
         #self.esbmc_nolibrary = "--no-library"
         self.esbmc_nolibrary = ""
         self.esbmc_extra_op = ""
-        self.esbmc_solver_op = "--z3"
+        self.esbmc_solver_op = "--boolector"
         self.use_counter_example = False
         # k-induction options
         self.esbmc_basecase_op = "--base-case"
         self.esbmc_forwardcond_op = "--forward-condition"
-        self.esbmc_inductivestep_op = "--inductive-step --no-slice --show-counter-example"
+        self.esbmc_inductivestep_op = "--inductive-step "
 
-        self.dldv_error = " -DLDV_ERROR=ERROR "
-        self.dassert = " -D_Bool=int "
+        self.dldv_error = " "
+        self.dassert = " "
         self.no_bounds_check = ""
         self.no_pointer_check = ""
         self.no_div_by_zero_check = ""
         self.no_assertions = ""
         self.memory_leak_check = ""
         self.no_inductive_step = False
-        self.quiet = " --quiet "
```

Comment:
If you remove this option, then ESBMC will print the entire log file in the output, which might be too big.

Reply:
The option is set back to the code after tested.